### PR TITLE
feat: update gha workflows

### PR DESCRIPTION
**Pull request by mikejgray**
Preparation for stable releases

## Summary by CodeRabbit

- **New Features**
	- Enhanced GitHub Actions workflows for license testing, stable release proposals, and alpha build publishing, improving clarity and functionality.
	- Introduced new jobs for managing version updates and intent testing, expanding testing capabilities.
	- Updated skill metadata to reflect new ownership and repository location.

- **Bug Fixes**
	- Streamlined workflow triggers and naming conventions for improved clarity and performance.

- **Chores**
	- Consolidated workflow sources under a single organization for better management.

### CODE REVIEW

File: skill.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "title": "Laugh",
-    "url": "https://github.com/OpenVoiceOS/ovos-skill-laugh",
+    "url": "https://github.com/mikejgray/skill-laugh",
```
Comment (NeonDaniel) [inline skill.json:3]: This doesn't match the PR repo

File: skill.json [hunk 2]
```diff
@@ -48,8 +48,8 @@
         "- [SoundBible](http://soundbible.com/suggest.php?q=laugh&x=0&y=0)",
         "- [FreeSound](https://freesound.org/search/?q=female+evil+laugh)"
     ],
-    "skillname": "ovos-skill-laugh",
-    "authorname": "OpenVoiceOS",
+    "skillname": "skill-laugh",
```
Comment (NeonDaniel) [inline skill.json:51]: Is this an intentional name change? It will be a breaking change for config/settings

File: skill.json (hunk 1 above)
Comment (mikejgray) [reply skill.json:3]: Correct, because it ran on my fork, but as soon as it merges this will be corrected automatically

File: skill.json (hunk 2 above)
Comment (mikejgray) [reply skill.json:51]: No, it's based on the forked repo's name. Same deal as above

File: skill.json (hunk 1 above)
Comment (NeonDaniel) [reply skill.json:3]: :facepalm: not sure why I read this as being in `setup.py`/package data. makes complete sense now

**Review by NeonDaniel — APPROVED** [review]
Looks good to me

**Pull request closed without merging**